.github/workflows/cross.yml

```diff
@@ -134,14 +134,23 @@ jobs:
       - name: Install cargo-xwin and bindgen-cli
         run: cargo install cargo-xwin bindgen-cli
       - name: cargo xwin build for `x86_64-pc-windows-msvc`
-        run: cargo xwin build -p aws-lc-rs --release --target x86_64-pc-windows-msvc
+        run: cargo xwin build -p aws-lc-rs --release --all-targets --target x86_64-pc-windows-msvc
       - name: cargo xwin build for `aarch64-pc-windows-msvc`
-        run: cargo xwin build -p aws-lc-rs --release --target aarch64-pc-windows-msvc
+        run: cargo xwin build -p aws-lc-rs --release --all-targets --target aarch64-pc-windows-msvc
 
-  aws-lc-rs-windows-mingw:
+  aws-lc-rs-windows-cross:
     if: github.repository_owner == 'aws'
-    name: x86_64-pc-windows-gnu
+    name: ${{ matrix.target }}
     runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - 'x86_64-pc-windows-gnu'
```

Comment:
Still, it makes no sense to `-Clink-arg=/WX` if we're using the GNU toolchain...

Reply:
Yeah, I've done some refactoring in the latest commit to separate the Windows CI jobs building for "gnu" versus "msvc".